lib/nerves_hub_link/connection.ex

```diff
@@ -1,124 +0,0 @@
-defmodule NervesHubLink.Connection do
-  @moduledoc """
-  Agent used to keep the simple state of the devices connection
-  to [nerves-hub.org](https://www.nerves-hub.org).
-
-  The state is a tuple where the first element is an atom of `:connected` or
-  `:disconnected` and the second element is the value of `System.monotonic_time/1`
-  at the time of setting the new state.
-
-  In practice, this state is set anytime the device connection to
-  [nerves-hub.org](https://www.nerves-hub.org) channel changes.
-  Likewise, it is set after a HTTP request fails or succeeds. This makes it
-  useful when you want to consider the connection to
-  [nerves-hub.org](https://www.nerves-hub.org) as part of the overall health
-  of the device and perform explicit actions based on the result, such as using
-  the Erlang [:heart](http://erlang.org/doc/man/heart.html) module to force a
-  reboot if the callback check fails.
-
-  ```
-  # Set a callback for heart to check every 5 seconds. If the function returns anything other than
```

Comment:
Oh wow. I don't remember this advice for some reason, but it seems best that it is being deleted. A better check would be if there's been no connection a all for the past 15 minutes or hour or whatever, and then to tie it in with firmware validation.